include/common/firmware/tim7.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
```

Comment:
This lives in common, but the .c file lives in gantry. I think it's a pretty good idea - it's the static callbacks again basically - but if we're going to do it this way I think the header here should not specify tim7, just specify an api for starting and stopping a general purpose timer, like
```
void motor_hardware_init(); // implies MX_GPIO_Init() and MX_TIM7_Init() and TIM7_Init()
void motor_hardware_start(); // implies TIM7_Start_IT
void motor_hardware_stop(); // implies TIM7_Stop_IT
```

and probably should have some way of indicating what the interrupt should call.